inference-engine/src/mkldnn_plugin/mkldnn_node.cpp

```diff
@@ -1281,19 +1283,19 @@ MKLDNNNode* MKLDNNNode::NodesFactory::create(const std::shared_ptr<ngraph::Node>
         }
     }
 
-    if (newNode == nullptr) {
-        try {
-            std::unique_ptr<MKLDNNNode> ol(new MKLDNNReferenceNode(op, eng, w_cache, errorMessage));
-            if (ol != nullptr && ol->created(extMgr))
-                newNode = ol.release();
-        } catch (const InferenceEngine::Exception& ex) {
-            if (ex.getStatus() != NOT_IMPLEMENTED) {
-                throw;
-            } else {
-                errorMessage += getExceptionDescWithoutStatus(ex);
-            }
-        }
-    }
+    // if (newNode == nullptr) {
+    //     try {
+    //         std::unique_ptr<MKLDNNNode> ol(new MKLDNNReferenceNode(op, eng, w_cache, errorMessage));
+    //         if (ol != nullptr && ol->created(extMgr))
+    //             newNode = ol.release();
+    //     } catch (const InferenceEngine::Exception& ex) {
+    //         if (ex.getStatus() != NOT_IMPLEMENTED) {
+    //             throw;
+    //         } else {
+    //             errorMessage += getExceptionDescWithoutStatus(ex);
+    //         }
+    //     }
+    // }
```

Comment:
Why?

Reply:
it's needed to have correct performance report (without performance drop when unsupported node fallback on ngraph implementation)

Reply:
comment added